vuepress/docs/next/docs/curate/ecr-bundle-details.md

```diff
@@ -123,11 +123,18 @@ Here is an example of a plugin descriptor:
         role: manage-users
       - clientId: realm-management
         role: view-users
+    environmentVariables: # a set of environment variables that can be set inline or reference secrets in the namespace
```

Comment:
i think we've been capitalizing "Secret(s)" everywhere because the general meaning is different than the K8s def

Reply:
Just a note, these are K8s Secrets not a different kind of secret.